reactor-core/src/main/java/reactor/core/scheduler/BoundedElasticScheduler.java

```diff
@@ -160,7 +170,11 @@ public void init() {
 				// The executor was most likely shut down in parallel.
 				// If the state is SHUTDOWN - it's ok, no eviction schedule required;
 				// If it's running - the other thread did a restart and will run its own schedule.
```

Comment:
restart is mentioned in this comment, this can be amended

Reply:
Even though restarts are discouraged, they can still be the cause for this situation.